t/core/config.t

```diff
@@ -38,7 +38,7 @@ __DATA__
 GET /t
 --- response_body
 etcd host: http://127.0.0.1:2379
-first plugin: "real-ip"
+first plugin: "ai"
```

Comment:
we can change the priority of ai plugin
we should avoid to change this test case

Reply:
If it is not necessary, I am going to change it in the next PR